### ISSUE TITLE
How to build for 64 bit JVM.

### ISSUE THREAD

**Issue by gkaushal** [apangin/jattach#33]
I built the windows code but when i run the tool i get the following error.
"Cannot attach 64-bit process to 32-bit JVM"

What can be done to solve it?

**Comment by apangin** [comment]
The error message speaks for itself. 32-bit `jattach.exe` is needed to attach to 32-bit JVM processes, and 64-bit `jattach.exe` to attach to 64-bit JVM processes.

Use x86 compiler target to build 32-bit jattach.

Note that [release page](https://github.com/apangin/jattach/releases) already contains prebuilt binaries, including `jattach32.exe` for 32-bit JVMs.

**Comment by heatzync** [comment]
There is no `jattach32.exe` prebuilt binary available under the [v2.0 release tag](https://github.com/apangin/jattach/releases/tag/v2.0).

The release notes do however mention:
> Support 32-bit JVM on Windows

I therefore tried the `jattach.exe` prebuilt binary, but it still results in: `Cannot attach 64-bit process to 32-bit JVM`

I also tried the `jattach32.exe` prebuilt binary available under the [v1.5 release tag](https://github.com/apangin/jattach/releases/tag/v1.5). This seems to be working better, but it gets stuck here:
```
PS C:\Users\Blah\Downloads> .\jattach32.exe 7108 jcmd  "help -all"
Connected to remote process
Response code =
```

I'm unable to type anything into the console. The cursor just keeps blinking. I can only stop it by pressing `Ctrl+C`

**Comment by apangin** [comment]
@heatzync jattach32.exe is now included in 2.1 release.
